src/Symfony/Component/Messenger/EventListener/StopWorkerOnTimeLimitListener.php

```diff
@@ -22,9 +22,9 @@
  */
 class StopWorkerOnTimeLimitListener implements EventSubscriberInterface
 {
-    private int $timeLimitInSeconds;
-    private ?LoggerInterface $logger;
-    private float $endTime = 0;
+    protected int $timeLimitInSeconds;
+    protected ?LoggerInterface $logger;
```

Comment:
The first two are injected in the constructor, so I don't see the need to expose them.